### ISSUE TITLE
[BUG] Consecutive tables get merged into one table

### ISSUE THREAD

**Issue by cuonghuunguyen** [RedBug312/markdown-it-multimd-table#34]
**Description**
When parsing 2 or more consecutive tables, they will be merged into one table with multiple `tbody`

**Example code**
```md
| --- | --- |
| 1   | 2   |
| 3   |  4   |

| --- | --- |
| 5   | 6   |
| 7   | 8    |
```

Expected output
```html
<table>

<tbody>
<tr>
<td>1</td>
<td>2</td>
</tr>
<tr>
<td>3</td>
<td>4</td>
</tr>
</tbody>
</table>
```

Actual output
```html
<table>
<tbody>
<tr>
<td>1</td>
<td>2</td>
</tr>
<tr>
<td>3</td>
<td>4</td>
</tr>
</tbody>
<tbody>
<tr>
<td>---</td>
<td>---</td>
</tr>
<tr>
<td>5</td>
<td>6</td>
</tr>
<tr>
<td>7</td>
<td>8</td>
</tr>
</tbody>
</table>
```

**Comment by redbug312** [comment]
Hi, this is expected behavior as defined in [MultiMarkdown](https://fletcher.github.io/MultiMarkdown-6/syntax/tables.html):
> You can create multiple `<tbody>` tags (for HTML) within a table by having a single empty line between rows of the table.

But this rule seems to have confused some (#18).
I'm considering add an option like `multitbody: false` to disable this.

**Comment by cuonghuunguyen** [comment]
Hi, Thank you for your very quick response. That would be great to have that feature. For now, I think I should apply the workaround to have 2 or more line breaks between tables. Do you have any better workarounds for the moment (e.g. overriding rules)? I would be appreciated to hear that.

**Comment by redbug312** [comment]
Yes. For now, the best workaround is to introduce more empty lines.

Adding a table caption at the bottom also indicates the end of tables, but this introduces conflict against common standards again.

**Comment by redbug312** [comment]
Closed for version 4.2.0.
